django_stomp/execution.py

```diff
@@ -41,6 +80,12 @@ def start_processing(
     broker_host_to_consume_messages: Optional[str] = None,
     broker_port_to_consume_messages: Optional[int] = None,
 ) -> Optional[Listener]:
+    global _listener, _gracefully_shutdown
```

Comment:
**thought:** I think using global variables is OK in our context. One thread will be handling the `start_processing` method. However, more than one may be created inside the listener to consume the messages (through the prefetch size parameter) and send heartbeats.